sos/plugins/openstack_nova.py

```diff
@@ -89,7 +93,12 @@ def setup(self):
             self.var_puppet_gen + "_placement/etc/httpd/conf.d/",
             self.var_puppet_gen + "_placement/etc/httpd/conf.modules.d/*.conf",
             self.var_puppet_gen + "_placement/etc/my.cnf.d/tripleo.cnf",
-            self.var_puppet_gen + "/../memcached/etc/sysconfig/memcached"
+            self.var_puppet_gen + "/../memcached/etc/sysconfig/memcached",
+            self.var_puppet_gen + "_libvirt/etc/libvirt/",
+            self.var_puppet_gen + "_libvirt/etc/my.cnf.d/tripleo.cnf",
+            self.var_puppet_gen + "_libvirt/etc/nova/migration/"
+            "authorized_keys",
```

Comment:
I guess this contains just some public keys that can be shared (but no secret there).

Reply:
yes, the authorized_keys only have pub keys in.

Reply:
Are those keys generated by or embedded in the product, or are they privately generated by the user?

Remember that some organisations consider IP addresses to be "confidential" (we don't scrub to that level in `sos` but there are 3rd party tools that do this).

If the public data also winds up in logs then it's less of a problem but it's not so simple as "they are public keys -> it's fine".

Reply:
they get generated by the triplo installer. if the pub key is too sensitive data I am ok to remove that information and we collect it on a case by case situation. should we do that?

I just checked the info on the a non container env and we collect there the full migration dir, which also has the private key . I think that is an issue which we should fix. we definitely need to exclude the etc/nova/migration/identity file.

Reply:
just for reference the /etc/nova/migration subdir got added just recently and it seems we collect everything under /etc/nova . thats why we have the identity file now

Reply:
If it's generated by the product and the user has no (normal/direct) control over it then that's fine - it's just product internal debug data. It's only if the users can supply their own keys that it becomes a concern.